pulsar/manager_endpoint_util.py

```diff
@@ -85,9 +90,14 @@ def submit_job(manager, job_config):
             )
 
         if job_config is not None:
+            job_id = job_config["job_id"]
             job_directory = job_config["job_directory"]
             jobs_directory = os.path.abspath(os.path.join(job_directory, os.pardir))
             command_line = command_line.replace('__PULSAR_JOBS_DIRECTORY__', jobs_directory)
+            command_line = command_line.replace(os.path.join(jobs_directory, _get_galaxy_job_id(job_id)), job_directory)
+            for action in remote_staging['setup']:
+                if 'action' in action.keys() and 'contents' in action['action']:
+                    action['action']['contents'] = action['action']['contents'].replace(os.path.join(jobs_directory, _get_galaxy_job_id(job_id)), job_directory)
```

Comment:
Thank you so much for the contribution! I like parts of it but I am working so hard on making Pulsar's handling of inputs, paths, and command-line generation more structured - just rewriting everything after the fact on the Pulsar side like this seems really error prone and heading in the opposite direction. Let me think about this and the underlying issue - but if you can avoid using ``__PULSAR_JOBS_DIRECTORY__`` I would recommend it.